tembo-cli/src/cmd/apply.rs

```diff
@@ -561,19 +578,25 @@ fn get_postgres_config_cloud(instance_settings: &InstanceSettings) -> Vec<PgConf
 
 fn get_extensions(
     maybe_extensions: Option<HashMap<String, tembo_config::Extension>>,
-) -> Vec<Extension> {
+) -> Result<Vec<Extension>, anyhow::Error> {
     let mut vec_extensions: Vec<Extension> = vec![];
 
     if let Some(extensions) = maybe_extensions {
         for (name, extension) in extensions.into_iter() {
-            let mut vec_extension_location: Vec<ExtensionInstallLocation> = vec![];
-
-            vec_extension_location.push(ExtensionInstallLocation {
-                database: Some("postgres".to_string()),
-                schema: None,
-                version: None,
-                enabled: extension.enabled,
-            });
+            let version = Runtime::new()
```

Comment:
the typical approach is just to make the parent function async, then use await.

Reply:
Yup, I can do that but then I will have to go all the way to the top function and add async/await for all. Do you think that's better?